.github/workflows/e2e-fulltests-ci.yml

```
@@ -13,6 +13,10 @@ on:
         type: string
         description: If testing a PR, this must be specified as well.
         required: false
+      ROLLINGRELEASE_FROM_TAG:
```

nit: Can we have this as `ROLLING_RELEASE_<...>` (same with others)?
```suggestion
      ROLLING_RELEASE_FROM_TAG:
```